ddtrace/appsec/processor.py

```diff
@@ -28,9 +28,9 @@
 log = get_logger(__name__)
 
 
-def _no_cookies(data):
+def _transform_headers(data):
     # type: (Dict[str, str]) -> Dict[str, str]
-    return {key: value for key, value in data.items() if key.lower() not in ("cookie", "set-cookie")}
+    return {key.lower(): value for key, value in data.items() if key.lower() not in ("cookie", "set-cookie")}
```

Comment:
```suggestion
    normalized = {}
    for header, value in data.items():
        header = header.lower()
        if header in ("cookie", "set-cookie"):   # TODO: Move this tuple to a frozenset ?
            continue
        normalized[header] = value
    return normalized
```

1-line is clever, but seeing `key.lower()` twice here makes my eye twitch 🤣